gammapy/background/make.py

```diff
@@ -46,6 +51,11 @@ def make_bg_cube_model(observation_table, fits_path, method='default'):
         Path to the data files.
     method : {'default', 'michi'}, optional
         Bg cube model calculation method to apply.
+    do_not_force_mev_units : bool, optional
+        Set to `True` to use the same energy units as the energy
+        binning for the bg rate. (Only applicable to the *default*
+        method; the *michi* always enforces `MeV` in the bg rate
+        units.)
```

Comment:
Well, actually the whole idea is to be **LESS** flexible, since we are enforcing some particular units.

In addition, the `michi` method should always use these units, since we want it to be as close to @mimayer's results as possible, in order to compare, so I think is is fine how it is.

The user can always unset this forcing of the units, by setting the varaible to `True`, in which case, the same units as the ones defined by the binning will be used (the most natural choice). If he wants something else, he will have to do it manually.

So, I'm leaving this as it is.